devsetup/scripts/crc-setup.sh

```diff
@@ -71,3 +71,14 @@ sudo update-ca-trust
 if [ $export_path == 1 ]; then
     echo "WARNING: you must add ~/bin in your PATH in order to access to crc binary"
 fi
+
+# apply post-config tweaks
+oc patch etcd cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableEtcd": true}}}' --type=merge
+oc patch authentications.operator.openshift.io cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer": true}}}' --type=merge
+oc scale --replicas=1 ingresscontroller/default -n openshift-ingress-operator
+oc scale --replicas=0 deployment.apps/console -n openshift-console
```

Comment:
I agree, most of the times you probably don't want/need the console, but maybe we want to make this optional. could also to default to disable it